omod/src/main/java/org/openmrs/module/htmlformentryui/page/controller/htmlform/ViewEncounterWithHtmlFormPageController.java

```diff
@@ -26,6 +28,19 @@ public void get(@RequestParam("encounter") Encounter encounter,
 
         patient.setPatient(encounter.getPatient());
 
+        AdministrationService administrationService = Context.getAdministrationService();
+        
+        String customPrintModule = administrationService.getGlobalProperty("htmlformentryui.customPrintModule");
+        String customPrintUrl = administrationService.getGlobalProperty("htmlformentryui.customPrintUrl");
+        String customPrintTarget = administrationService.getGlobalProperty("htmlformentryui.customPrintTarget");
```

Comment:
We should use UI Framework naming conventions here:

"customPrintModule" should be "customPrintProvider"
"customPrintUrl" should be "customPrintPage" or "customPrintPageName"

Reply:
Latest commit makes this change